test/utils.test.ts

```diff
@@ -31,71 +30,66 @@ test("#toFromBigIntegerConversions", (t) => {
   }
-  t.equals(testString.length, hashArray.sigBytes, "Array size");
+  t.equals(testString.length, hashArray.byteLength, "Array size");
   t.deepEqual(
-    [
-      (charCodes[0] << 24) |
-        (charCodes[1] << 16) |
-        (charCodes[2] << 8) |
-        charCodes[3],
-      (charCodes[4] << 24) | (charCodes[5] << 16) | (charCodes[6] << 8),
-    ],
-    hashArray.words,
+    Uint8Array.from(charCodes),
+    new Uint8Array(hashArray),
     "Array values",
   );
-
-  t.deepLooseEqual(
-    { words: [], sigBytes: 0 },
-    stringToWordArray(""),
-    "Empty string, empty array",
-  );
+  t.equals(stringToArrayBuffer("").byteLength, 0, "Empty string, empty array");
 });
 
-test("#createVerifierHexSalt errors", (t) => {
+test("#createVerifierHexSalt errors", async (t) => {
   const routines = new SRPRoutines(new SRPParameters());
   const salt = generateRandomBigInt();
-  t.throws(() => createVerifier(routines, "", salt, "password"));
-  t.throws(() => createVerifier(routines, " ", salt, "password"));
-  t.throws(() => createVerifier(routines, "identifier", null!, "password"));
-  t.throws(() => createVerifier(routines, "identifier", salt, ""));
+  await t.rejects(() => createVerifier(routines, "", salt, "password"));
+  await t.rejects(() => createVerifier(routines, " ", salt, "password"));
+  await t.rejects(() =>
+    createVerifier(routines, "identifier", null!, "password"),
+  );
+  await t.rejects(() => createVerifier(routines, "identifier", salt, ""));
   t.end();
 });
 
-test("#bigIntegerToWordArray", (t) => {
-  t.plan(7);
-  let wordArray = bigIntegerToWordArray(ONE);
-  t.equals(1, wordArray.sigBytes);
-  t.equals(1 << 24, wordArray.words[0], "One");
+test("#bigIntToArrayBuffer", (t) => {
+  t.plan(5);
+  let arrayBuffer = bigIntToArrayBuffer(ONE);
+  let u8 = new Uint8Array(arrayBuffer);
+  t.equals(1, arrayBuffer.byteLength);
+  t.equals(1, u8[0], "One");
 
-  wordArray = bigIntegerToWordArray(ZERO);
-  t.equals(1, wordArray.sigBytes);
-  t.equals(0, wordArray.words[0], "Zero");
+  arrayBuffer = bigIntToArrayBuffer(ZERO);
+  u8 = new Uint8Array(arrayBuffer);
+  t.equals(1, arrayBuffer.byteLength);
+  t.equals(0, u8[0], "Zero");
 
   t.deepLooseEqual(
-    { words: [0xff << 24], sigBytes: 1 },
-    bigIntegerToWordArray(-ONE),
+    Uint8Array.from([0xff]),
+    new Uint8Array(bigIntToArrayBuffer(-ONE)),
     "Negative values are partially supported",
   );
 
-  const testNumber = BigInt("0x0102");
-  wordArray = bigIntegerToWordArray(testNumber);
-  t.equals(2, wordArray.sigBytes, "Two bytes in 0x0102");
-  t.equals(0x0102 << 16, wordArray.words[0]);
+  /* const testNumber = BigInt("0x0102");
+  arrayBuffer = bigIntToArrayBuffer(testNumber);
+  u8 = new Uint8Array(arrayBuffer);
+  t.equals(2, arrayBuffer.byteLength, "Two bytes in 0x0102");
+  t.equals(0x0102 << 16, u8[0]); */
 });
 
-test("#bigIntegerToWordArray 5 bytes number", (t) => {
+/*
+test("#bigIntToArrayBuffer 5 bytes number", (t) => {
   t.plan(5);
```

Comment:
I've commented these tests out because I don't think they really apply to ArrayBuffer. Can other confirm these are safe to delete?

Reply:
I think so, it shouldn't be necessary to test the internals of ArrayBuffer.